src/Filesystem/DTO/FileSize.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace ILIAS\Filesystem\DTO;
```

Comment:
@klees I think it was my fault to name them like this. The general idea at least how I learned it is the transfer data between different layers of an application with the help of dtos.
The FileSize has some small logic which is not really a dto class (I will move the class in another package.) The problem is ILIAS has no layers in general which makes it difficult to tell which one are dtos.

You also probably noticed the sentence "A value object is not a DTO. The two terms have been conflated by Java developers in the past." I need to disagree with that it is not only in the past because I am a Java developer. xD

We could also call them value object but that would also be wrong because the value objects are immutable and the "dto" class has at least in Java terms no such restriction.

Personally, I would like to use the term dto because those are mutable or immutable data classes which are used to transfer data between two services. Furthermore, these classes never contain test worthy code and are considered equal in terms of data and not the reference. But I can't deny that I am somewhat biased by my Java experience.